styles/tah-pf2e.css

```diff
@@ -92,3 +92,8 @@ img[src="systems/pf2e/icons/mdi/thrown.svg"].tah-list-image {
     min-width: 16px;
     width: 16px;
 }
+
+.tah-group[data-has-image="true"][data-level="4"] {
+    position: relative;
+    right: 21px;
+}
```

Comment:
Each group is intentionally indented and this change will impact not only the Strikes but any group that sits at level 4 and has an image.

This could instead be a group setting within Token Action HUD Core to allow TAH system modules to remove indentation from specific groups as desired.

Reply:
True, didn't think about how it would affect other groups, will revert this change altogether and think of a better way to handle this.